discovery-provider/alembic/trigger_sql/handle_track.sql

```diff
@@ -9,17 +9,30 @@ begin
   insert into aggregate_track (track_id) values (new.track_id) on conflict do nothing;
   insert into aggregate_user (user_id) values (new.owner_id) on conflict do nothing;
 
+  -- increment or decrement?
+  if TG_OP = 'UPDATE' then
```

Comment:
there are only a few places where we do in-place UPDATE (like marking unavailable).

Normally it'd be an `INSERT` and we'd have to explicitly go select `old_row` right?